locales-src/asrouter.ftl

```diff
@@ -81,3 +81,29 @@ cfr-whatsnew-button =
   .tooltiptext = What’s New
 
 cfr-whatsnew-panel-header = What’s New
+
+## Recommended features
+
+cfr-feature-type-recommended = Recommended Feature
+
+## Bookmark Sync
+
+cfr-sync-bookmarks-header = Get this bookmark on your phone
+cfr-sync-bookmarks-body = Take your bookmarks, passwords, history and more everywhere you’re signed into { -brand-product-name }.
+cfr-sync-bookmarks-cta-button = Turn on { -sync-brand-short-name }
```

Comment:
The existing ids are `cfr-doorhanger-extension-ok-button` and `cfr-doorhanger-pintab-ok-button`, so these all should probably start with `cfr-doorhanger-` and the buttons should be `ok` instead of `cta`